src/main/java/ch/njol/skript/expressions/ExprRandom.java

```diff
@@ -47,13 +47,24 @@
 		"give a random item out of all items to the player"})
 @Since("1.4.9")
 public class ExprRandom extends SimpleExpression<Object> {
+
 	static {
 		Skript.registerExpression(ExprRandom.class, Object.class, ExpressionType.COMBINED, "[a] random %*classinfo% [out] of %objects%");
 	}
 	
 	@SuppressWarnings("null")
 	private Expression<?> expr;
 	
+	@Override

```

Comment:
Why did you move this above, it's always `init` -> `get` -> `isSingle` and `getReturnType` -> `toString`, unless there's other stuff

Reply:
No it's not. It is always the way I put it.

It's the order asending from the simple expression class.

Reply:
But that isn't the convention so can you please move them where they belong?